WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -19,10 +20,11 @@ public partial class ShowQrCameraDialogViewModel : DialogViewModelBase<string?>
 	[AutoNotify] private string _errorMessage = "";
 	[AutoNotify] private string _qrContent = "";
 
-	public ShowQrCameraDialogViewModel(Network network)
+	public ShowQrCameraDialogViewModel(UiContext context, Network network)
```

Comment:
you can either add UiContext and keep the ctor public (what you did here) or just simply make the ctor private so the UiContext will be added in a public ctor that is source generated.

So IMO making the ctor private would be enough.

Reply:
I would do that, but as I wrote in the description the generator for whatever reason doesn't hook it up:

```diff
-       public ShowQrCameraDialogViewModel(UiContext context, Network network)
+       private ShowQrCameraDialogViewModel(Network network)
        {
                _qrReader = new(network);
                SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
-               UiContext = context;
        }
```
```
./WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs(92,41): error CS1729: 'ShowQrCameraDialogViewModel' does not contain a constructor that takes 2 arguments
./WalletWasabi.Fluent/ViewModels/Dialogs/AddressEntryDialogViewModel.cs(50,41): error CS1729: 'ShowQrCameraDialogViewModel' does not contain a constructor that takes 2 arguments
```

So that public constructor is a workaround to unblock me.

Reply:
@soosr we need to improve the source gen to also consider calls to methods such as `Close()` as a dependency on `UiContext`, because internally they actually depend on it.

I'll open an issue about this.